cli/pkg/backend/aws/backend.go

```diff
@@ -281,7 +285,7 @@ func (b *Backend) PrintLogs(ctx context.Context, stackName string, serviceName s
 	return b.ComputeBackend.PrintLogs(ctx, stackName, serviceName, opts...)
 }
 
-func (b *Backend) RunTask(ctx context.Context, taskDefArn string, launchType config.LaunchType) error {
+func (b *Backend) RunTask(ctx context.Context, taskDefArn string, launchType util.LaunchType) error {
```

Comment:
we need the variable names to be more descriptive than a letter

Reply:
This construct refers to a binding type struct that will have these functions assigned to. Happy throughout the codebase has shortened var names for the grouping struct as it is not a traditional variable. It's not the name, but the type of the grouping struct that matters, and it is descriptive (`Backend`).

Reply:
That only works for functions with limited lines of code; using cryptic names will affect readability as they grow.  Case in point with the ab situation, where I had to expand the file to figure out what ab is.
https://readlearncode.com/code-and-stuff/clean-code-variables/

Reply:
`ab` is not a variable; it's a pretty standard convention in golang to shorten conflicting or lengthy package names.

Reply:
Here's an example from kubernetes code: https://github.com/kubernetes/kubernetes/blob/fcfe5dfc2177320833caa451b401f2d1dc5668e7/pkg/securitycontext/util.go#L20

Reply:
As this code is written by convention, and is consistent across packages, we are not sacrificing readability. Go is a language of limited verbosity, unlike Java.